pokemongo_bot/__init__.py

```diff
@@ -578,32 +579,24 @@ def find_close_cells(self, lat, lng):
         return map_cells
 
     def _setup_logging(self):
-        # log settings
-        # log format
+        log_level = logging.ERROR
 
         if self.config.debug:
             log_level = logging.DEBUG
-            logging.getLogger("requests").setLevel(logging.DEBUG)
-            logging.getLogger("websocket").setLevel(logging.DEBUG)
-            logging.getLogger("socketio").setLevel(logging.DEBUG)
-            logging.getLogger("engineio").setLevel(logging.DEBUG)
-            logging.getLogger("socketIO-client").setLevel(logging.DEBUG)
-            logging.getLogger("pgoapi").setLevel(logging.DEBUG)
-            logging.getLogger("rpc_api").setLevel(logging.DEBUG)
-        else:
-            log_level = logging.ERROR
-            logging.getLogger("requests").setLevel(logging.ERROR)
-            logging.getLogger("websocket").setLevel(logging.ERROR)
-            logging.getLogger("socketio").setLevel(logging.ERROR)
-            logging.getLogger("engineio").setLevel(logging.ERROR)
-            logging.getLogger("socketIO-client").setLevel(logging.ERROR)
-            logging.getLogger("pgoapi").setLevel(logging.ERROR)
-            logging.getLogger("rpc_api").setLevel(logging.ERROR)
-
-        logging.basicConfig(
-            level=log_level,
-            format='%(asctime)s [%(name)10s] [%(levelname)s] %(message)s'
-        )
+
+        logging.getLogger("requests").setLevel(log_level)
+        logging.getLogger("websocket").setLevel(log_level)
+        logging.getLogger("socketio").setLevel(log_level)
+        logging.getLogger("engineio").setLevel(log_level)
+        logging.getLogger("socketIO-client").setLevel(log_level)
+        logging.getLogger("pgoapi").setLevel(log_level)
+        logging.getLogger("rpc_api").setLevel(log_level)
+
+        if self.config.logging_clean and not self.config.debug:
+            formatter = Formatter(fmt='[%(asctime)s] %(message)s', datefmt='%H:%M:%S')
```

Comment:
@nanogirl21 Do you think it could be possible to have the choice of what to show in the settings ? I like seeing the name, but I totally understand that’s superfluous to most user. Having the choice with this setting by default seems a good compromise to me.